apps/cowswap-frontend/src/common/pure/TransactionErrorContent/index.tsx

```diff
@@ -40,21 +41,23 @@ const AlertIcon = styled(AlertTriangle)`
   margin-bottom: 15px;
 `
 
+const BackButtonStyled = styled(BackButton)`
+  margin-right: -22px;
+`
+
 export interface TransactionErrorContentProps {
   message: JSX.Element | string
   onDismiss(): void
-  mode?: 'screen' | 'modal'
+  isScreenMode?: boolean
```

Comment:
Same here. I'd prefer to have a standard name for this.

Reply:
Fixed: https://github.com/cowprotocol/cowswap/pull/3756/commits/08afda61eca55c270ff0dac10aa8a68c5a88ef6d